lib/vsc/mympirun/rm/sched.py

```diff
@@ -266,8 +258,11 @@ def set_mpinodes(self):
         if getattr(self.options, 'double', False):
             res = self.nodes * 2
         else:
-            for uniquenode in nub(self.nodes):
-                res.extend([uniquenode] * self.mpiprocesspernode)
+            if getattr(self.options, 'hybrid', None):
```

Comment:
yeah, but it defaults to `None` when it isn't set explicitly, so this check will be False